### PR TITLE
Key list backend ready

### PR DESCRIPTION
**Pull request by Chernyshelly**

### CODE REVIEW

File: server/WebApi/Controllers/KeysController.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    using Application.Interfaces;
+    using Application.ViewModels;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Logging;
+
+    [ApiController]
+    [Route("api/[controller]")]
+    public class KeysController : ControllerBase
+    {
+        private readonly ILogger<KeysController> logger;
```
Comment (jskonst) [inline server/WebApi/Controllers/KeysController.cs:14]: also please rename this one to _logger

File: server/Infrastructure/Repository/TeacherRepository.cs [hunk 2]
```diff
@@ -22,6 +22,11 @@ public Teacher InsertTeacher(Teacher teacher)
             return entity.Entity;
         }
 
+        public Teacher GetTeacherById(int id)
+        {
+            return context.Teachers.AsNoTracking().FirstOrDefault(t => t.Id == id);
```
Comment (jskonst) [inline server/Infrastructure/Repository/TeacherRepository.cs:27]: this changes are not related to the task, I guess

Comment (Chernyshelly) [reply server/Infrastructure/Repository/TeacherRepository.cs:27]: It was made during the process and will be useful soon, there is no reason to delete it and rewrite it again
